sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/optimizer/TransposeWindowSuite.scala

```diff
@@ -90,14 +90,16 @@ class TransposeWindowSuite extends PlanTest {
   }
 
   test("don't transpose two adjacent windows with intersection of partition and output set") {
-    val query = testRelation
-      .window(Seq(('a + 'b).as('e), sum(c).as('sum_a_2)), partitionSpec3, Seq.empty)
-      .window(Seq(sum(c).as('sum_a_1)), Seq(a, 'e), Seq.empty)
+    if (!conf.ansiEnabled) {
+      val query = testRelation
+        .window(Seq(('a + 'b).as('e), sum(c).as('sum_a_2)), partitionSpec3, Seq.empty)
```

Comment:
what's wrong with ansi mode here?

Reply:
string + string

Reply:
I don't think this test case want to test type coercion. Can we add explicit cast or use other string expressions like `concat('a, 'b)`?